packages/core/src/indexing/types.ts

```diff
@@ -0,0 +1,66 @@
+import type { StreamID } from '@ceramicnetwork/streamid'
+import type { DID } from 'dids'
+
+export interface IndexStreamArgs {
+  streamID: StreamID
+  model: StreamID
+  controller: DID
+  lastAnchor: Date | null
+  content: Record<string, any>
```

Comment:
I'd probably leave out `content` and `relationFields` entirely for now, we can add them in the future when we actually support doing anything with them